esphome/components/htu21d/sensor.py

```diff
@@ -54,3 +68,43 @@ async def to_code(config):
     if CONF_HUMIDITY in config:
         sens = await sensor.new_sensor(config[CONF_HUMIDITY])
         cg.add(var.set_humidity(sens))
+
+    if CONF_HEATER in config:
+        sens = await sensor.new_sensor(config[CONF_HEATER])
+        cg.add(var.set_heater(sens))
+
+
+@automation.register_action(
+    "htu21d.set_heater_level",
+    SetHeaterLevelAction,
+    cv.Schema(
+        {
+            cv.GenerateID(): cv.use_id(HTU21DComponent),
+            cv.Required(CONF_LEVEL): cv.templatable(cv.int_),
+        }
+    ),
```

Comment:
This allows for yaml such as this if there is only 1 htu21d platform in yaml as it can auto populate the id.
```yaml
- htu21d.set_heater_level: xx
```

```suggestion
    cv.maybe_simple_value(
        {
            cv.GenerateID(): cv.use_id(HTU21DComponent),
            cv.Required(CONF_LEVEL): cv.templatable(cv.int_),
        },
        key=CONF_LEVEL,
    ),
```